ncbimeta/NCBImeta.py

```diff
@@ -388,7 +388,8 @@ def UpdateDB(table, output_dir, database, email, search_term, table_columns, log
 
             if column_payload[0] == "XPATH":
                 column_payload.remove("XPATH")
-                NCBImetaUtilities.adv_xml_search(working_root, column_payload, column_name, column_dict)
+                column_payload_xpath = column_payload[0]
```

Comment:
Do you think we should add null checking here? 
A user could potentially configure a column name as such:
```
MyColumnName : XPATH
```
Or we could use something like the `''.join(my_list)` method to always return a string regardless if the list is empty.